content/updates/terraform-use-cases.md

```diff
@@ -0,0 +1,201 @@
+Hopefully, your takeaway is the same thing we preach to everyone we work with: if you want a solid platform, use IaC for _everything_.
+
+### Why Aren’t People Using Terraform for Everything?
+
+There are a few reasons why engineers might not be using Terraform to deploy resources like Git repositories, user management, or monitoring. One common reason we often see is that they simply don't think to do it, as they may be more focused on implementing everything else from scratch. In the case of Git repositories, engineers might view it as a chicken-and-egg problem, where they manually create repositories before deploying code, rather than automating the entire process with Terraform. Additionally, time constraints and pressure to deliver quickly can lead engineers to stick with their tried-and-tested workflows instead of exploring new approaches like using Terraform to handle it all.
+
+### Team member account management and Role Based Access Control
+
+Managing user accounts and roles across multiple SaaS products can be a real headache, especially when those products don't support Single Sign-On (SSO). It's a task that often falls on the shoulders of senior leadership and engineering teams, eating up hours of their time that could be better spent on more strategic initiatives. Fortunately, TF can make this process much less painful.
+
+Before we look at an example, it's worth noting that many SaaS vendors seem to have missed the memo on the importance of SSO for security. As highlighted by [The SSO Wall of Shame](https://sso.tax/), many vendors offer SSO but treat it as a premium feature, bundling it with expensive "Enterprise" pricing tiers or charging multiple times the base product price for the functionality. This practice disincentivizes the use of SSO and encourages poor security practices, especially for smaller organizations that may not have the budget for the higher-tier plans.
+
+In a typical software engineering organization, there are likely several service platforms that require managing users and access levels. A common, bare-minimum stack might include AWS, GitHub, CloudFlare, and Datadog for starters. For teams without the benefit of SSO, access to these services is a cumbersome task to manage manually. Each time a team member joins or leaves the organization means one person needs to log in to all of these platforms and add or remove that team member.  With TF, you can centralize users and identity management and make life much easier (and more secure).
+
+For AWS users, AWS offers IAM Identity Center (formerly AWS SSO), which simplifies AWS account and role management. But for organizations that haven't jumped on the SSO bandwagon yet, or are using a mix of services with and without SSO support, TF can standardize onboarding and offboarding regardless of vendor. By defining users and roles in a `team.yaml` file, TF Root Modules can automatically create and manage these entities across all the different platforms you're using.
```

Comment:
```suggestion
For AWS users, AWS offers [IAM Identity Center](https://aws.amazon.com/iam/identity-center/) (formerly AWS SSO), which simplifies AWS account and role management. But for organizations that haven't jumped on the SSO bandwagon yet, or are using a mix of services with and without SSO support, TF can standardize onboarding and offboarding regardless of vendor. By defining users and roles in a `team.yaml` file, TF Root Modules can automatically create and manage these entities across all the different platforms you're using.
```
Heard outbound links are good for SEO 😄

Reply:
Haha I know nothing about SEO, so I'm happy someone is thinking of it!